src/lib/icons/IconDown.svelte

```diff
@@ -0,0 +1,20 @@
+<!-- source: DFINITY foundation -->
+<script lang="ts">
+  import { DEFAULT_ICON_SIZE } from "$lib/constants/constants";
+  export let size = `${DEFAULT_ICON_SIZE}px`;
+</script>
+
+<svg
+  height={size}
+  width={size}
+  viewBox="0 0 20 20"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+  <path
+    fill-rule="evenodd"
+    clip-rule="evenodd"
+    d="M10.7604 0L9.35472 0L9.35459 14.1319L4.09146 8.51553L2.99693 9.54121L9.51014 16.4916L10.0574 17.0756L10.6046 16.4916L17.1027 9.55789L16.0082 8.53218L10.7603 14.132L10.7604 0ZM17.9998 18V19.5L2 19.4998V17.9998L17.9998 18Z"
+    fill="currentColor"
```

Comment:
Does it work if your replace `fill=none` with `fill="currentColor"` and remove this tag?

i.e. space one attribute

Reply:
Yes, done.